### ISSUE TITLE
Deployment issues

### ISSUE THREAD

**Issue by Han-code0819** [OpenDriveLab/PersFormer_3DLane#100]
Is there anyone deploy successfully? I cannot get onnx with below code:

"""outputs and tests onnx file"""
from models.PersFormer import PersFormer
import onnx
import torch
from utils.utils import *
from config import persformer_openlane, persformer_once
from mmcv.runner import load_checkpoint

checkpoint_path = '/path/model_best_epoch_55.pth.tar'
onnx_path = '/path/model_best_epoch_55.onnx'
batch = 16

parser = define_args() # args in utils.py
args = parser.parse_args()
persformer_openlane.config(args)

# model
model = PersFormer(args)
model.cuda()
load_checkpoint(model, checkpoint_path)
model.eval()

init_input = torch.rand(batch, 3, 1920, 1080).cuda()

# Export the model
torch.onnx.export(
    model,
    init_input,
    onnx_path,
    opset_version=11,
    input_names = ["input"],
    output_names = ["output0","output1","output2","output3","output4", "output5"],
)
 
# check onnx model
onnx_model = onnx.load(onnx_path)

try:
    onnx.checker.check_model(onnx_model)
except Exception:
    print('ONNX Model Incorrect')
else:
    print('ONNX Model Correct')
    
onnx.save(onnx_model, onnx_path)

![image](https://user-images.githubusercontent.com/112684346/218666839-5861f538-41ff-47e7-9330-00af8e2594c3.png)
![image](https://user-images.githubusercontent.com/112684346/218666978-b62f6625-d0e8-4d45-81c1-80e4899b2391.png)

In "class RefPntsNoGradGenerator", "__init__" has M with a specific tensor, but in "forward", M == None.
M: normalized transformation matrix between image view and IPM

Could anyone please give me some instructions
, or you have getonnx.py directly?

**Comment by RicardLee** [comment]
This problem may be caused by pytorch and posting this qusetion on other professional communities, such as stack overflow, may be helpful.

**Comment by huangmozhi9527** [comment]
Hi, PersFormer takes a front-view image and transformation matrice as input. In your getonnx code, the transformation matrice is ignored, causing the error.
